README.md

```diff
@@ -1576,6 +1576,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
       <td align="center"><a href="https://github.com/AndreaCUS"><img src="https://avatars.githubusercontent.com/u/44421522?v=4?s=100" width="100px;" alt="AndreaCUS"/><br /><sub><b>AndreaCUS</b></sub></a><br /><a href="#design-AndreaCUS" title="Design">🎨</a> <a href="https://github.com/Ebazhanov/linkedin-skill-assessments-quizzes/commits?author=AndreaCUS" title="Code">💻</a></td>
       <td align="center"><a href="https://www.linkedin.com/in/simran-chaurasia-003"><img src="https://avatars.githubusercontent.com/u/88017902?v=4?s=100" width="100px;" alt="Simran"/><br /><sub><b>Simran</b></sub></a><br /><a href="#design-simransia" title="Design">🎨</a> <a href="https://github.com/Ebazhanov/linkedin-skill-assessments-quizzes/commits?author=simransia" title="Code">💻</a></td>
       <td align="center"><a href="https://www.linkedin.com/in/prashant-bhapkar-064a78145/"><img src="https://avatars.githubusercontent.com/u/60314439?v=4?s=100" width="100px;" alt="Prashant Bhapkar"/><br /><sub><b>Prashant Bhapkar</b></sub></a><br /><a href="#design-Prashant-Bhapkar" title="Design">🎨</a> <a href="https://github.com/Ebazhanov/linkedin-skill-assessments-quizzes/commits?author=Prashant-Bhapkar" title="Code">💻</a></td>
+      <td align="center"><a href="http://tsg405.github.io"><img src="https://avatars.githubusercontent.com/u/65300045?v=4?s=100" width="100px;" alt="TSG"/><br /><sub><b>TSG</b></sub></a><br /><a href="#design-TSG405" title="Coding">🎨</a> <a href="https://github.com/tsg405" title="Code">💻</a></td>
```

Comment:
```suggestion
```

do not rush, it should be automatically added by all-contributors command. Please remove for now